src/mmw/package.json

```diff
@@ -49,6 +49,8 @@
     "turf-random": "1.0.2",
     "underscore": "1.8.3",
     "watchify": "3.1.0",
-    "zeroclipboard": "2.2.0"
+    "zeroclipboard": "2.2.0",
+    "turf-intersect": "1.4.2",
```

Comment:
Consider keeping the packages in `package.json` sorted alphabetically, since that is the behavior of `npm install <packagename> --save`. Thus, this would be: 

```json
    "turf-area": "1.1.1",
    "turf-buffer": "1.0.4",
    "turf-erase": "1.3.2",
    "turf-intersect": "1.4.2",
    "turf-random": "1.0.2",
```